code/datums/ai/basic_mobs/basic_subtrees/speech_subtree.dm

```diff
@@ -36,7 +36,7 @@
 
 /datum/ai_planning_subtree/random_speech/cockroach
 	speech_chance = 5
-	emote_hear = list("chitters")
+	emote_hear = list("chitters.")
```

Comment:
does this actually need to be punctuated? Why?

Reply:
why does it need to be punctuated? I don't think we do this with emotes normally, and looking at even other emote_hears it's like a 50/50

Reply:
because it's ugly? improper? a sin? the only thing between us and anarchy?
would you prefer i add punctuation to all mob emotes that don't have them in this pr?

Reply:
You know the only fair path is to run an algorithm that removes punctuation from exactly half of all says.

Reply:
I just like, emotes aren't full sentences, so I don't think they need periods.
I don't care horribly, it just seems strange to target just this, and not all the other sorts of emote text. I don't get it.

Reply:
I only singled out this emote because I've seen it via ghost ears like 50 times booting maps on local

Reply:
admin powercreep, depressing